src/components/Button.js

```diff
@@ -44,12 +44,15 @@ const propTypes = {
     /** Call the onPress function when Enter key is pressed */
     pressOnEnter: PropTypes.bool,
 
-    /** Additional styles to add after local styles */
+    /** Additional styles to add after local styles. Applied to Pressable portion of button */
     style: PropTypes.oneOfType([
         PropTypes.arrayOf(PropTypes.object),
         PropTypes.object,
     ]),
 
+    /** Additional button styles. Specific to the OpacityView of button */
+    buttonStyles: PropTypes.arrayOf(PropTypes.object),
+
```

Comment:
I could name this something like innerStyles, but I figured buttonStyles makes more sense here. I realized that there's no way to put in a custom height/padding for buttons because the `styles` prop actually only affects the container/Pressable portion of the button, not the inner view.

Reply:
That seems odd to me because we pass `button.styles` directly to the `OpacityView` as well?

Reply:
We do pass the [`styles.button`](https://github.com/Expensify/App/blob/bfe3e1d4e391e867507045b8cfd582d570f76565/src/components/Button.js#L190) but not the styles itself. That only happens [now with this line](https://github.com/Expensify/App/blob/bfe3e1d4e391e867507045b8cfd582d570f76565/src/components/Button.js#L202).

The `additionalStyles` are getting added to the [Pressable part only](https://github.com/Expensify/App/blob/bfe3e1d4e391e867507045b8cfd582d570f76565/src/components/Button.js#L183).

Reply:
I think `containerStyles` and `innerStyles` makes a bit more sense here, but maybe I'm not entirely understanding your reasoning.

The `Pressable` is the container, and the `OpacityView` is the inner component. If I was looking at this component for the first time I think this makes more sense than `additionalStyles` and `buttonStyles`, since the whole component is called `Button`.

Reply:
This makes a lot of sense to me. Container styles really is the most accurate way to describe that for button. I think I wanted to avoid renaming the existing "styles" prop, but it isn't too difficult to see where `<Button` is referenced and just change the name there.